src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureColor {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, String color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + String.format("%.1f", getArea())
```

Comment:
Use the `name()` method to get the string representation of enum constants instead of `toString()` if `getColor()` returns an enum type. If `getColor()` returns a string, ensure that the color values are in English to maintain consistency and universality in your code.